shell/browser/api/electron_api_notification.h

```diff
@@ -67,18 +67,20 @@ class Notification : public gin::Wrappable<Notification>,
   void Close();
 
   // Prop Getters
-  std::u16string GetTitle() const;
-  std::u16string GetSubtitle() const;
-  std::u16string GetBody() const;
-  bool GetSilent() const;
-  bool GetHasReply() const;
-  std::u16string GetTimeoutType() const;
-  std::u16string GetReplyPlaceholder() const;
-  std::u16string GetUrgency() const;
-  std::u16string GetSound() const;
-  std::vector<electron::NotificationAction> GetActions() const;
-  std::u16string GetCloseButtonText() const;
-  std::u16string GetToastXml() const;
+  std::u16string get_title() const { return title_; }
```

Comment:
The `get_` prefix is usually omitted in Chromium's code, for example:
https://source.chromium.org/chromium/chromium/src/+/main:ui/views/view.h;l=542-549?q=views::View&ss=chromium

Also, it might be a good change to make the string getters return `const std::u16string&` instead.

Reply:
> The get_ prefix is usually omitted in Chromium's code, for example:

Good call. Removed in  7d91107,  38204e5, c84e5da

> Also, it might be a good change to make the string getters return `const std::u16string&` instead.

Also good call. Updated in c84e5da